saleor/graphql/api.py

```diff
@@ -140,11 +140,11 @@ class Query(graphene.ObjectType):
     vouchers = DjangoFilterConnectionField(
         Voucher, query=graphene.String(description=DESCRIPTIONS['product']),
         description="List of the shop\'s vouchers.")
-    shipping_method = graphene.Field(
-        ShippingMethod, id=graphene.Argument(graphene.ID),
-        description='Lookup a shipping method by ID.')
-    shipping_methods = DjangoFilterConnectionField(
-        ShippingMethod, description='List of the shop\'s shipping methods.')
+    shipping_zone = graphene.Field(
+        ShippingZone, id=graphene.Argument(graphene.ID),
+        description='Lookup a shipping zone by ID.')
+    shipping_zones = DjangoFilterConnectionField(
```

Comment:
I don't see a resolver for this, but there is `resolve_shipping_methods` left which seems unused.